apps/admin-portal/src/components/applications/wizard/application-create-wizard.tsx

```diff
@@ -287,7 +414,7 @@ export const ApplicationCreateWizard: FunctionComponent<ApplicationCreateWizardP
             delete application.inboundProtocolConfiguration.oidc;
         }
 
-        createNewApplication(ApplicationManagementUtils.prefixTemplateNameToDescription(application, template));
+        createNewApplication(ApplicationManagementUtils.prefixTemplateNameToDescription(application, selectedTemplate));
```

Comment:
Check if there's a line length warning here.

Reply:
length is fine here